app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Form";
+
+const NewContactPane = ({ showPane, setShowPane }) => {
```

Comment:
Go through the Slack thread. The component name and the file name have disparity.